javascript/conditonal-statement.md

```diff
@@ -0,0 +1,259 @@
+# CONDITIONAL STATEMENT IN JAVASCRIPT.
+
+Conditional statements are statements that perform particular tasks based on the given condition(s).
+
+## This project explains the following:
+
+- What conditional statements are.
+- The `if` statement.
+- The `if...else` statement.
+- Logical operators.
+- ternary operators.
+- Truthy and falsy.
+- Truthy and falsy assignment.
+- The `else...if` statement.
+- Switch keyword.
+
+## The `if` statement:
+
+Tasks are mostly carried out based on condition. For example, if you are hungry, then you will eat. If it rains, then you use your umbrella.
+
+The same thing applies to our code . in javascript we use the `if` statement to perform tasks based on conditions.
+
+```js
+let  bestScore = 40;
+if (bestScore <= 45);{
+console.log(" please play again")
+}
+\\ prints: Please play again.
```

Comment:
I don't think this is the right way to comment code. Can we use a backslash instead of this forward slash? See below:

```js
// RIGHT

\\ WRONG
```
_Please do this for all other occurrences._